artview/components/plot_radar.py

```diff
@@ -778,9 +778,10 @@ def _check_file_type(self):
     ########################
     def _quick_savefile(self, PTYPE=IMAGE_EXT):
         '''Save the current display via PyArt interface.'''
-        PNAME = self.display.generate_filename(
+        path = self.display.generate_filename(
             self.Vfield.value, self.Vtilt.value, ext=IMAGE_EXT)
-        print "Creating " + PNAME
+        self.canvas.print_figure(path, dpi=DPI)
```

Comment:
I would prefer ```imagename``` to ```path``` because this generates a filename and not a filepath+name.

On that note, can you change ```path``` in L783 to ```os.path.join(os.getcwd(), imagename)```?

The reason this was in there was to quickly save an image to current working directory instead of selecting some directory and entering a filename.